sinusbot_installer.sh

```diff
@@ -927,7 +924,7 @@ if [ "$YT" == "Yes" ]; then
     redMessage "Cronjob already set for YT-DL updater"!
   else
     greenMessage "Installing Cronjob for automatic YT-DL update..."
-    echo "0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename >/dev/null" >>/etc/cron.d/ytdl
+    printf "SHELL=/bin/sh\nPATH=/usr/local/bin/\n0 0 * * * $SINUSBOTUSER youtube-dl -U --restrict-filename > /dev/null 2>&1" >>/etc/cron.d/ytdl
```

Comment:
As @mxschmitt mentioned: Why are you setting SHELL and PATH? There is also no need for `printf`. Just use an absolute path:
`echo "0 0 * * * $SINUSBOTUSER /usr/local/bin/youtube-dl -U --restrict-filename > /dev/null 2>&1" >>/etc/cron.d/ytdl`

Reply:
it doesn't work, Cron clears the whole environment..
This what users get
 [CODE]/bin/sh: 1: youtube-dl: not found [/CODE]